Project-18-Food-order-app/src/main.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.jsx";
+import "./index.scss";
+import { CartContextProvider } from "./services/stores/CartContext";
+import { UserProgressProvider } from "./services/stores/UserProgress.jsx";
```

Comment:
@sknepali Make use of write imports in order as below
inbuilt lib imports
then local components/service imports
then styles at the end